Practical Example/RWTakeHome/RWTakeHome/Model/Tutorial.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Tutorial.swift
+//  RWTakeHome
+//
+//  Created by Nick Nguyen on 10/7/20.
+//
+
+import Foundation
+
+enum TutorialType: String {
+  case article
+  case video
+}
+
+struct Tutorial: Decodable {
+
+  var attributes: Attributes
+}
+
+struct DataTutorial: Decodable {
+  let tutorials: [Tutorial]
+
+  enum CodingKeys: String, CodingKey {
+    case tutorials = "data"
+  }
+}
+
+struct Attributes {
+  
+  let name: String
+  let artwork: URL
+  let description: String
+  let releaseDateString: String
+  let contentType: String
+
+  var releaseDate: Date {
+    let dateFormatter = DateFormatter()
+    dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+    dateFormatter.timeZone = TimeZone.autoupdatingCurrent
+    dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+    return dateFormatter.date(from: releaseDateString) ?? Date()
+  }
+
+  enum CodingKeys: String, CodingKey {
+    case name = "name"
+    case artwork = "card_artwork_url"
+    case description = "description_plain_text"
+    case releaseDate = "released_at"
+    case contentType = "content_type"
```

Comment:
Nice job using coding keys to make sure your variables are named according to Swift standards!